common/arg.cpp

```diff
@@ -376,6 +376,34 @@ static std::vector<ggml_backend_dev_t> parse_device_list(const std::string & val
     return devices;
 }
 
+static void add_rpc_devices(std::string servers) {
+    size_t pos = 0;
+    std::vector<std::string> rpc_servers;
+    while ((pos = servers.find(',')) != std::string::npos) {
```

Comment:
Btw, you can also use `string_split<std::string>(servers, ',')`

Reply:
fixed, thanks